toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -493,6 +494,15 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		return
 	}
 
+	if len(config.Users) > 0 || len(config.Groups) > 0 {
+		shadowUtilsInstalled := 0
+		shadowUtilsInstalled, err = TdnfInstallWithProgress(shadowUtilsPkg, installRoot, packagesInstalled, totalPackages, true)
+		if err != nil {
+			return
+		}
+		packagesInstalled += shadowUtilsInstalled
+	}
```

Comment:
Bit of a nit, and we don't do this for the filesystem above so... 🤷
Also not 100% sure this works (I think its good though...)
```suggestion
		// Remove shadow-utils from the list of packages to install
		packagesToInstall = sliceutils.FindMatches(packagesToInstall, func(pkg string) bool { return pkg != "shadow-utils" })

	}
```

Reply:
Applied this suggestion anyway and confirmed it works.